src/test/java/uk/gov/mca/beacons/service/accounts/AccountsControllerUnitTest.java

```diff
@@ -0,0 +1,115 @@
+package uk.gov.mca.beacons.service.accounts;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+
+import java.util.UUID;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.security.servlet.SecurityAutoConfiguration;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import uk.gov.mca.beacons.service.model.AccountHolder;
+
+@WebMvcTest(
+  controllers = AccountsController.class,
+  excludeAutoConfiguration = { SecurityAutoConfiguration.class }
+)
+@AutoConfigureMockMvc
+public class AccountsControllerUnitTest {
+
+  @Autowired
+  private MockMvc mvc;
+
+  @MockBean
+  private AccountsService accountService;
+
+  private UUID accountHolderId = UUID.fromString(
+    "432e083d-7bd8-402b-9520-05da24ad143f"
```

Comment:
Can use the static `UUID.random()`

Reply:
Discussed on the call - decided to keep the explicit example random UUID in there so the reader of the test sees the UUID, not just the method that generates it.